auth-api/src/auth_api/services/affiliation.py

```diff
@@ -203,10 +207,16 @@ def is_authorized(entity: Entity, pass_code: str) -> bool:
         return True
 
     @staticmethod
-    def create_new_business_affiliation(org_id,  # pylint: disable=too-many-arguments, too-many-locals
-                                        business_identifier=None, email=None, phone=None, certified_by_name=None,
+    def create_new_business_affiliation(affiliation_data: AffiliationData,  # pylint: disable=too-many-locals, too-many-branches, too-many-statements  # noqa: E501
```

Comment:
too many branches, means we should refactor into functions

Reply:
It was added in this PR, there was no too many branches code smell or linting issue before - I'm fairly positive